docker/transformers-pytorch-deepspeed-latest-gpu/Dockerfile

```diff
@@ -11,8 +11,9 @@ ARG REF=main
 RUN git clone https://github.com/huggingface/transformers && cd transformers && git checkout $REF
 RUN python3 -m pip install --no-cache-dir -e ./transformers[deepspeed-testing]
 
-RUN git clone https://github.com/microsoft/DeepSpeed && cd DeepSpeed && rm -rf build && \
-    DS_BUILD_CPU_ADAM=1 DS_BUILD_AIO=1 DS_BUILD_UTILS=1 python3 -m pip install -e . --global-option="build_ext" --global-option="-j8" --no-cache -v --disable-pip-version-check 2>&1
+# Install latest release PyTorch
+RUN python3 -m pip uninstall -y torch torchvision torchaudio
+RUN python3 -m pip install --no-cache-dir -U torch torchvision torchaudio
```

Comment:
won't the second command that includes `-U` take care of uninstall automatically?

Reply:
You caught me !

Reply:
there are some peculiar circumstances where an explicit uninstall is needed.

Reply:
for all the options please see: https://pytorch.org/get-started/locally/

Reply:
Regarding `--no-cache-dir`, I just copied it from other places. People on StackOverflow say this can reduce docker image.
@LysandreJik might have other insights here?

Reply:
no-cache-dir is to have a leaner image. It's already quite heavy with frameworks such as PyTorch which end up weighing 1GB+, skipping the cache ensures the download is as fast as it can be.